### PR TITLE
usb_negative: correct usb_reply_msg to validate usb_negative_test

### PR DESCRIPTION
**Pull request by ybduan**
1.remove the semicolon at the end of usb_reply_msg.
  usb_negative_test becomes invalid with this semicolon.
2.update usb_reply_msg with latest info.
3.remove the reboot step at the end of usb_negative_test.

ID: 1849539

Signed-off-by: ybduan <yduan@redhat.com>

### CODE REVIEW

**Comment by ybduan** [comment]
(1/2) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.usb.usb_host.usb_one_time.usb_negative_test.without_usb_hub.nec-xhci: PASS (35.89 s)
 (2/2) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.usb.usb_host.usb_one_time.usb_negative_test.without_usb_hub.qemu-xhci: PASS (28.78 s)

**Comment by ybduan** [comment]
Please help review this PR, thanks! @yanan-fu

File: qemu/tests/cfg/usb.cfg [hunk 1]
```diff
@@ -311,13 +311,15 @@
...
-                    usb_reply_msg = "Property 'usb-host.productid' doesn't take value"
+                    usb_reply_msg = "Property 'usb-host.productid' doesn't take value;"
                     usb_reply_msg += "Parameter 'productid' expects an int64 value or range;"
                     usb_reply_msg += "Parameter 'vendorid' expects an int64 value or range;"
                     usb_reply_msg += "productid out of range;"
                     usb_reply_msg += "vendorid out of range;"
                     usb_reply_msg += "Parameter 'productid' expects uint32_t;"
                     usb_reply_msg += "Parameter 'vendorid' expects uint32_t;"
+                    usb_reply_msg += "Parameter 'productid' expects uint64;"
+                    usb_reply_msg += "Parameter 'vendorid' expects uint64"
```
Comment (yanan-fu) [inline qemu/tests/cfg/usb.cfg:322]: Have you tested with the old version ?

Comment (ybduan) [reply qemu/tests/cfg/usb.cfg:322]: Yes, it will always be PASS.

Comment (yanan-fu) [reply qemu/tests/cfg/usb.cfg:322]: @ybduan i am puzzled, why both `uint32_t` and `uint64` all can pass ? Could you explain more ?

Comment (yanan-fu) [reply qemu/tests/cfg/usb.cfg:322]: Ok, thanks!

**Pull request merged**